students/hyeonwoo/users/views.py

```diff
@@ -62,10 +64,17 @@ def post(self, request):
             if not User.objects.filter(email = user_email).exists():
                 return JsonResponse({'MESSAGE' : 'INVALID_EMAIL'}, status = 401)
 
-            if User.objects.get(email = user_email).password != user_password:
+            user = User.objects.get(email = user_email)
+
+            encoded_user_password = user_password.encode('utf-8')
+            encoded_db_password = user.password.encode('utf-8')
+
+            if bcrypt.checkpw( encoded_user_password, encoded_db_password ):
```

Comment:
```suggestion
            if not bcrypt.checkpw( encoded_user_password, encoded_db_password ):
```
일치하지 않을 때를 조건으로 만들어서 else를 없애보세요.
else가 있다는 것은 코드 흐름상 생각할 것이 한 번 더 있다는 이야기 입니다. 당연히 else가 필요한 순간에는 사용해야 되지만, 안 쓸수 있으면 최대한 안 쓰는 방향으로 코드를 진행해 보세요.